mijia/src/bluetooth.rs

```diff
@@ -14,7 +14,7 @@ use std::fmt::{self, Display, Formatter};
 use std::str::FromStr;
 use std::sync::Arc;
 
-/// Opaque identifier for a bluetooth device which the system knows about.
+/// Opaque identifier for a Bluetooth device which the system knows about.
```

Comment:
/// An opaque identifier for the device, including a reference to which adapter it was
    /// discovered on. This can be used to connect to it.

add some of this information here too

Reply:
Done.